src/Components/ProvisioningWizard/steps/ReservationProgress/index.js

```diff
@@ -91,7 +91,7 @@ const ReservationProgress = ({ setLaunchSuccess }) => {
 
   // polling request
   const { data: polledReservation } = useQuery(['reservation', reservationID], () => fetchReservation(reservationID), {
-    enabled: !!reservationID && currentStep < steps.length && !currentError,
+    enabled: !!reservationID && !currentError,
```

Comment:
the `refetch` stops when the success/error flags are set, so no need to have this condition here. just caused this edge case